src/doxygen.cpp

```diff
@@ -12693,6 +12693,14 @@ void generateOutput()
          Portable::getSysElapsedTime()
         );
     g_s.print();
+    bool restore=FALSE;
+    if (Debug::isFlagSet(Debug::Time))
```

Comment:
the same check is already done at line 12689 it seems, so I would expect this guard is always true and the code can be simplified.

Reply:
Indeed, fixed.